Formula/ntopng.rb

```diff
@@ -1,9 +1,18 @@
 class Ntopng < Formula
   desc "Next generation version of the original ntop"
   homepage "http://www.ntop.org/products/ntop/"
-  url "https://downloads.sourceforge.net/project/ntop/ntopng/ntopng-2.2.tar.gz"
-  sha256 "4fccfc9e9f333addcd3c957b4520c471117bc2df5655d6eabf328c7385fb255e"
-  revision 1
+
+  stable do
+    url "https://github.com/ntop/ntopng/archive/2.4.tar.gz"
+    sha256 "86f8ed46983f46bcd931304d3d992fc1af572b11e461ab9fb4f0f472429bd5dd"
+
+    resource "nDPI" do
+      # tip of 1.8-stable branch; four commits beyond the 1.8 tag
+      url "https://github.com/ntop/nDPI.git",
+        :tag => "1.8-stable",
```

Comment:
`colortail` and `x264` apparently, but I only checked the stable spec and only the primary resource, ignoring all resources specified via `resource do`:

```
brew for-each-formula 'puts "#{f.full_name}: #{f.stable.specs.inspect}" if f.stable && f.stable.specs.key?(:revision) && !f.stable.specs.key?(:tag)'
```

I can't think of a reason why this wouldn't work if the default branch isn't `master`, but I haven't actually investigated that. What problem do you foresee?